diff -r -u freeradius-server-3.0.1.bak/Make.inc.in freeradius-server-3.0.1/Make.inc.in
--- freeradius-server-3.0.1.bak/Make.inc.in	2014-01-17 11:20:48.599733053 -0500
+++ freeradius-server-3.0.1/Make.inc.in	2014-01-17 14:37:33.739853796 -0500
@@ -47,7 +47,7 @@
 
 CC		= @CC@
 RANLIB		= @RANLIB@
-IMACROS		= -imacros ${top_srcdir}/src/freeradius-devel/build.h -imacros ${top_srcdir}/src/freeradius-devel/autoconf.h -imacros ${top_srcdir}/src/freeradius-devel/features.h
+IMACROS		= -include ${top_srcdir}/src/freeradius-devel/build.h -include ${top_srcdir}/src/freeradius-devel/autoconf.h -include ${top_srcdir}/src/freeradius-devel/features.h
 INCLUDE		= -I${top_srcdir} -I${top_srcdir}/src
 CFLAGS		= $(IMACROS) $(INCLUDE) -std=c99 -fno-strict-aliasing @CFLAGS@
 CPPFLAGS	= @CPPFLAGS@
Only in freeradius-server-3.0.1: Make.inc.in~
